sale_blanket_order/models/blanket_orders.py

```diff
@@ -327,7 +327,7 @@ def action_cancel(self):
 
     def action_view_sale_orders(self):
         sale_orders = self._get_sale_orders()
-        action = self.env.ref("sale.action_orders").read()[0]
+        action = self.env.ref("sale.action_orders").sudo().read()[0]
```

Comment:
@PicchiSeba There is a core function to do this, look for '_for_xml_id()'

Reply:
Thanks!